src/cloudant/database.py

```diff
@@ -299,3 +305,151 @@ def __iter__(self, remote=True):
                     yield doc
 
             raise StopIteration
+
+    def security_document(self):
+        """
+        _security_document_
+
+        Fetch the security document for this database
+        which contains information about who the database
+        is shared with
+
+        GET _api/v2/db/<dbname>/_security
+
+        """
+        resp = self._r_session.get(self.security_url)
+        resp.raise_for_status()
+        return resp.json()
+
+    def share_database(self, username, reader=True, writer=False, admin=False):
```

Comment:
There are other properties too, like _replicator and I can't remember what else -- might want to ask a dbcore engineer.